infrastructure/Cdk/Cdk.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
7.0?

Reply:
Lambda doesn't have native support for .NET 7. They have custom runtimes but I've never used them. We could try out the custom runtimes later, but just wanted to get it up and running for now.

Reply:
#10